### PR TITLE
adding tags field to booking complex field

### PR DESCRIPTION
**Pull request by rsomavarapu-sift**
https://github.com/SiftScience/sift-java/compare/add_tags_field_to_booking?expand=1

adding tags field to booking complex field

**Purpose**

Add `$tags` optional field to `$booking` object.

**Technical overview**

Update `booking.java` with `$tags` field.

**Testing Plan**

- Updated unit tests
- Send sample events using SiftClient and verify the changes on the console

**Deployment**
Publish to Maven Central and create a GitHub release.

### CODE REVIEW

**Comment by rsomavarapu-sift** [comment]
**create_order event**

Request Body: https://gist.github.com/rsomavarapu-sift/53991df76b3a77a3951d342f11a4ce98
Response Body: https://gist.github.com/rsomavarapu-sift/247f98e9eb34d4b272ae55c986bccdf1

Screen Shot 2019-07-31 at 10 16 45 AM
Screen Shot 2019-07-31 at 10 19 22 AM
Screen Shot 2019-07-31 at 10 19 33 AM
Screen Shot 2019-07-31 at 10 21 21 AM
Screen Shot 2019-07-31 at 10 21 25 AM

**Comment by rsomavarapu-sift** [comment]
**update_order event**

Request Body: https://gist.github.com/rsomavarapu-sift/c80fe0f8e9f12957ecaa4d6d0b5e0723
Response Body: https://gist.github.com/rsomavarapu-sift/13bcd5d9de0469c21c7d6ec9edd765b4
Screen Shot 2019-07-31 at 10 25 07 AM


Screen Shot 2019-07-31 at 10 25 33 AM
Screen Shot 2019-07-31 at 10 25 54 AM
Screen Shot 2019-07-31 at 10 26 03 AM
Screen Shot 2019-07-31 at 10 26 08 AM

File: src/test/java/com/siftscience/TestUtils.java [hunk 1]
```diff
@@ -129,7 +145,7 @@ static Guest sampleGuest1() {
                 .setLoyaltyProgram("skymiles")
                 .setLoyaltyProgramId("PSOV34DF")
                 .setPhone("1-415-555-6040")
-                .setEmail("jdeo@domain.com");
+                .setEmail("jdoe@domain.com");
```
Comment (jkbach) [inline src/test/java/com/siftscience/TestUtils.java]: nice :)

File: src/test/java/com/siftscience/TestUtils.java [hunk 2]
```diff
@@ -1,6 +1,14 @@
 package com.siftscience;
 
-import com.siftscience.model.*;
```
Comment (jkbach) [inline src/test/java/com/siftscience/TestUtils.java]: Good catch.

**Pull request merged**